tests/e2e/online_runner.py

```
@@ -0,0 +1,83 @@
+    # Let ZMQ subscribers connect
+    time.sleep(0.2)
+
+    # Parse and send events
+    csv_path = dataset["base"] / dataset["csv"]
+    events, delays_ns = parse_csv(csv_path, stream_info)
+
+    streamer = pycer.PyStreamer("tcp://localhost", stream_listener_port)
+
+    has_timing = any(d > 0 for d in delays_ns)
+    total = len(events)
+    for i, (event, delay_ns) in enumerate(zip(events, delays_ns)):
```

Ruff (0.15.4)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)